### ISSUE TITLE
"et" and similar cases in the entry

### ISSUE THREAD

**Issue by abausi** [SChAth/dillmann#135]
in case we have an entry like "ሲኖዶስ et ሴኖዶስ" also "et" will be visualized, creating a lot of noise in the results: how can we mend this?

**Comment by abausi** [comment]
@PietroLiuzzo

**Comment by PietroLiuzzo** [comment]
I have just added this which I though would be a feature, in response to your notification of the problem about searching ሴኖዶስ. 
There are in fact terms where the lemma has "ሲኖዶስ et ሴኖዶስ". what generated the first error was exactely the presence of two geez words in the lemma. I then fixed it by reporting both with et as in the source. 
which kind of search gets which kind of noise in the results?
